src/lib.rs

```diff
@@ -93,7 +95,22 @@ thread_local! {
 }
 
 lazy_static! {
-    static ref DEPENDENCY_GRAPH: Mutex<DiGraph<usize>> = Default::default();
+    static ref DEPENDENCY_GRAPH: Mutex<DiGraph<usize, MutexEdge>> = Default::default();
+}
+
+#[derive(Clone)]
+struct MutexEdge(Arc<Backtrace>);
+
+impl MutexEdge {
+    fn capture() -> Self {
+        MutexEdge(Arc::new(Backtrace::capture()))
```

Comment:
`Backtrace::capture` is a no-op by default. Does it make sense to force capturing the backtrace anyway depending on `debug_assertions`?